src/exercises/Congratulations.js

```diff
@@ -32,9 +32,15 @@ export default function Congratulations({
           😳 Pay more attention to
           <br />
           <br />
-          {incorrectBookmarks.map((each) => (
-            <Word bookmark={each} api={api} />
-          ))}
+          {incorrectBookmarks
+            .reduce(
```

Comment:
Here you're trying to remove duplicates in the list?

Because then it's more readable and also more concise if you do 

    x = [... new Set(x)]

Reply:
Right! Thanks. I was so stuck on doing everything in the return function that I blinded myself. I've adjusted this in the cb1c4ed99ce57bc94c2ab8171963b8c1a3bb7d61 commit and already did it also for the correct answers array as duplicates might become an issue when the match exercise comes into play.